tools/provision/lib.sh

```diff
@@ -368,6 +368,13 @@ function install_libaptpkg() {
   fi
 }
 
+function link_openssl_pkgconfig_files() {
+  BREW_PREFIX=$(brew --prefix)
```

Comment:
This will run on existing OpenSSL install via brew, if you expand the context: lines 393-414